modules/manage/pages/security/iam-roles.adoc

```diff
@@ -135,4 +144,7 @@ After satisfying the prerequisites for your cloud platform, edit the Redpanda cl
 
 | `gcp_instance_metadata`
 | For a VM running on GCP, or for Google Kubernetes Engine (GKE), use the instance metadata API from GCP.
+
+| `azure_vm_instance_metadata`
+| For a VM running on Azure, use the Instance Metadata Service (IMDS) from Azure.
```

Comment:
@JakeSCahill should I go ahead and add `azure_aks_oidc_federation` here? Let me know if you prefer to do it in your PR

Reply:
Here is good, thanks Kat. I think it makes sense for me to add K8s in your branch.